test/unit/cpu/main-nested.cxx

```diff
@@ -296,15 +296,15 @@ struct PolLTimesA {
   typedef NestedPolicy<ExecList<seq_exec, seq_exec, seq_exec, seq_exec>> EXEC;
 
   // psi[direction, group, zone]
-  typedef RAJA::View<double, Layout<RAJA::Index_type, PERM_IJK, IDirection, IGroup, IZone>>
+  typedef RAJA::TypedView<double, PermutedLayout<PERM_IJK>, IDirection, IGroup, IZone>
```

Comment:
Are the labels 'PSI_VIEW', 'PHI_VIEW', 'ELL_VIEW',  etc. sufficiently generic or are they specific to the particular code they were originally developed for?